scripts/common/git_helpers.py

```diff
@@ -17,3 +19,35 @@ def git_current_branch():
 
 def git_commit_hash(ref: str = 'HEAD'):
     return run_git_command(['git', 'rev-parse', '--verify', ref])
+
+
+class FileMismatchError(Exception):
+    def __str__(self):
+        return f"Diff mismatch:\n {self.args[0]}"
+
+
+def diff(file_a: Path, file_b: Path):
+    """
+    Returns with no output if there are no differences between the two inputs.
+    Otherwise (i.e. if the process exits with a non-zero exit code) throws FileMismatchError.
+    """
+
+    if which("git") is None:
+        raise RuntimeError("git not found.")
+
+    try:
+        subprocess.check_output(
+            f"""\
+            git diff \
+                --color \
```

Comment:
Nice. I wanted to complain that this may not work on all platforms, but I guess it should be fine since it's `git diff` and not the crappy `diff` included by default with macOS, which does not seem to support `--color`. Maybe we should later switch to using git for diffing in our Bash scripts too.